src/sentry/integrations/tasks/slack/send_notifications_on_activity.py

```diff
@@ -10,15 +10,15 @@
 from sentry.utils import metrics
 
 _default_logger = logging.getLogger(__name__)
-_SUCCESS_METRIC = "sentry.integrations.slack.tasks.send_notifications_on_activity.send_activity_notifications.success"
-_FAILURE_METRIC = "sentry.integrations.slack.tasks.send_notifications_on_activity.send_activity_notifications.failure"
+_SUCCESS_METRIC = "sentry.integrations.tasks.slack.send_notifications_on_activity.send_activity_notifications.success"
+_FAILURE_METRIC = "sentry.integrations.tasks.slack.send_notifications_on_activity.send_activity_notifications.failure"
 _TASK_QUEUED_METRIC = (
-    "sentry.integrations.slack.tasks.send_notifications_on_activity.activity_created_receiver"
+    "sentry.integrations.tasks.slack.send_notifications_on_activity.activity_created_receiver"
 )
 
 
 @instrumented_task(
-    name="sentry.integrations.slack.tasks.send_activity_notifications_to_slack_threads",
+    name="sentry.integrations.slack.send_activity_notifications_to_slack_threads",
```

Comment:
I'm pretty sure changing the `name` of a task is a breaking change -- old code won't be able to queue up for the new code and new code won't be able to queue up for the old code (in the case of a rollback)

Reply:
Yes, changing celery task names will cause failures. I'll restore this and the other changed task names, as we don't want to drop any tasks during deploys. A rollback would cause failures with the changes as they stand right now, as we're introducing new task names that wouldn't be present in a previous release 🤔

Reply:
probably need a multi-stage rollout then (1) introduce new tasks (2) cut over to new tasks (3) remove old tasks

Reply:
Yeah. We'll split this up into multiple stages so that we can bridge rollbacks and forwards deploys.